stdlib/nodejs/io.rb

```diff
@@ -1,3 +1,64 @@
+class IO
+
+  @__fs__ = node_require :fs
+  `var __fs__ = #{@__fs__}`
+
+  attr_reader :eof
+  attr_reader :lineno
+
+  def initialize
+    @eof = false
+    @lineno = 0
+  end
+
+  def read(length = nil)
+    if @eof
+      ''
+    else
+      if length.nil?
+        res = `__fs__.readSync(#{@fd}).toString()`
```

Comment:
@elia `readSync` is not working, can I replace this line with `__fs__.readFileSync(#{@path}).toString()` ?